### ISSUE TITLE
is.oin examples and functionality broken

### ISSUE THREAD

**Issue by pitkant** [rOpenGov/sweidnumbr#124]
Currently ```is.oin``` has the following examples:

```
library(sweidnumbr)
ex_oin <- c("556000-4615", "232100-0156", "802002-4280", "8020024280", "AA2002-4280")
is.oin(ex_oin)
```

When run, the test returns false. This is understandable as ```is.oin``` only tests if the elements in a vector inherit the class "oin":

```
function (oin) 
{
    inherits(oin, "oin")
}
```

This works:

```
is.oin(as.oin(ex_oin))
[1] TRUE
```

Maybe this additional ```as.oin``` step should be added to the examples?

Another problem with the function is that it returns a single TRUE/FALSE output when the man pages promise to "test which elements in a text vector that contains organization identity number" and to return a "logical vector indicating if the elements can be an organization identity number". Now the output is not a vector and it does not tell which elements are TRUE and which are FALSE.

**Comment by MansMeg** [comment]
This is not a bug, but needs to be fixed in a 1.0 version. It just ceck if the class is oin (as is.xxx genererally works in R). So the docs needs to be clarified.

**Comment by pitkant** [comment]
Yes, I agree that much confusion can be resolved by improving the documentation. 

What about as.oin returning a single TRUE/FALSE for a vector containing multiple OIN numbers? Shouldn't it return a vector of same length, testing all OIN numbers individually?

**Comment by MansMeg** [comment]
No, that is oin_ctrl (testing the control number). as.oin also returns NA for incorrectly formatted oins when parsing a character or numeric vector.

**Comment by MansMeg** [comment]
Now fixed in v1.5.0
